src/patternfly/components/Drawer/drawer.scss

```diff
@@ -33,58 +43,174 @@ $pf-c-drawer__panel--width-list: (25, 33, 50, 66, 75);
     --pf-c-drawer__panel-body--PaddingTop: var(--pf-c-drawer__panel-body--md--PaddingTop);
     --pf-c-drawer__panel-body--PaddingRight: var(--pf-c-drawer__panel-body--md--PaddingRight);
     --pf-c-drawer__panel-body--PaddingBottom: var(--pf-c-drawer__panel-body--md--PaddingBottom);
     --pf-c-drawer__panel-body--PaddingLeft: var(--pf-c-drawer__panel-body--md--PaddingLeft);
+    --pf-c-drawer__content-body--m-padding--PaddingTop: var(--pf-c-drawer__content-body--m-padding--md--PaddingTop);
+    --pf-c-drawer__content-body--m-padding--PaddingRight: var(--pf-c-drawer__content-body--m-padding--md--PaddingRight);
+    --pf-c-drawer__content-body--m-padding--PaddingBottom: var(--pf-c-drawer__content-body--m-padding--md--PaddingBottom);
+    --pf-c-drawer__content-body--m-padding--PaddingLeft: var(--pf-c-drawer__content-body--m-padding--md--PaddingLeft);
+  }
+
+  @media screen and (min-width: $pf-global--breakpoint--xl) {
+    --pf-c-drawer__panel--FlexBasis: var(--pf-c-drawer__panel--xl--FlexBasis);
+    --pf-c-drawer__panel--MinWidth: var(--pf-c-drawer__panel--xl--MinWidth);
   }
 
   // Box shadow
   --pf-c-drawer__panel--BoxShadow: none;
-  --pf-c-drawer--m-expanded__panel--BoxShadow: var(--pf-global--BoxShadow--md-left);
-  --pf-c-drawer--m-expanded--m-panel-left__panel--BoxShadow: var(--pf-global--BoxShadow--md-right);
+  --pf-c-drawer--m-expanded__panel--BoxShadow: var(--pf-global--BoxShadow--lg-left);
+  --pf-c-drawer--m-expanded--m-panel-left__panel--BoxShadow: var(--pf-global--BoxShadow--lg-right);
 
   // Divider
-  --pf-c-drawer--m-divider__panel--after--Width: var(--pf-global--BorderWidth--sm);
-  --pf-c-drawer--m-divider--m-expanded__panel--after--BackgroundColor: var(--pf-global--BorderColor--100);
+  --pf-c-drawer__panel--after--Width: var(--pf-global--BorderWidth--sm);
+  --pf-c-drawer__panel--after--BackgroundColor: transparent;
+  --pf-c-drawer--m-inline--m-expanded__panel--after--BackgroundColor: var(--pf-global--BorderColor--100);
+
+  // Simple list
+  --pf-c-drawer--c-simple-list__title--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-drawer--c-simple-list__title--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-drawer--c-simple-list__title--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-drawer--c-simple-list__title--PaddingLeft: var(--pf-global--spacer--lg);
+
+  // Simple list item
+  --pf-c-drawer--c-simple-list__item-link--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-drawer--c-simple-list__item-link--PaddingLeft: var(--pf-global--spacer--lg);
+
+  // Action
+  --pf-c-drawer__action--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-drawer__panel-body--c-drawer__action--MarginRight: calc(var(--pf-global--spacer--md) * -1);
 
   display: flex;
+  flex-direction: column;
+  min-width: var(--pf-c-drawer__panel--MinWidth);
   height: 100%;
   overflow-x: hidden;
+
+  .pf-c-simple-list__title {
+    --pf-c-simple-list__title--PaddingTop: var(--pf-c-drawer--c-simple-list__title--PaddingTop);
```

Comment:
I wonder if we can build this in to the simple list itself or maybe create some sort of `drawer__panel-nav` element or something like that, that is used to create space for some sort of list like this inside of it, similar to this in the sidebar that spaces the nav component from the top.

<img width="439" alt="Screen Shot 2020-02-07 at 5 26 08 PM" src="https://user-images.githubusercontent.com/35148959/74073203-10f87e00-49cf-11ea-9cee-ef96e5d6ede3.png">

Then we don't have to re-style the simple list in the drawer, since I presume most drawers don't have a simple list, and it creates a dependency. Also that could add padding to the bottom so we don't end up with this lack of space at the end of the list that looks off

<img width="259" alt="Screen Shot 2020-02-07 at 5 28 08 PM" src="https://user-images.githubusercontent.com/35148959/74073255-4735fd80-49cf-11ea-9bf1-51c74eaed567.png">

Reply:
Also that would allow someone to use the simple list somewhere else in the drawer in another context without getting these styles inadvertently, since the current styling will apply to all simple lists in a drawer.